### PR TITLE
Update readme with links to the new Tables SDK

### PR DESCRIPTION
**Pull request by ramya-rao-a**
Changes:

- We are close to releasing the Tables SDK, so updating the README of the azure-storage package with the right pointers.
- Also, took this opportunity to add the suggested standard text we use for older packages that are being replaced

cc @joheredi, @XiaoningLiu, @EmmaZhu, @vinjiang

### CODE REVIEW

**Review by EmmaZhu — APPROVED** [review]

**Comment by ramya-rao-a** [comment]
@EmmaZhu I don't have merge rights in this repo, please consider merging the PR now that it has gotten approval

**Review by XiaoningLiu — APPROVED** [review]

**Comment by ramya-rao-a** [comment]
Thank You!

**Pull request merged**
